config/initializers/devise.rb

```diff
@@ -266,7 +266,7 @@
   # config.navigational_formats = ['*/*', :html]
 
   # The default HTTP method used to sign out a resource. Default is :delete.
-  config.sign_out_via = :delete
+  config.sign_out_via = :get
```

Comment:
I see you added link to sign out with `:delete` method. Thus you don't have to change this file.

Reply:
This feature don't work without Jquery. I removed param from the link